ci/build.sh

```diff
@@ -18,6 +18,39 @@ function get_branch_to_checkout() {
     || echo "${TRAVIS_BRANCH}"
 }
 
+function bundle_fluentd_plugins() {
+  local VERSION="${1}"
+
+  if [[ "${VERSION}" == "" ]] ; then
```

Comment:
One could also use `local VERSION="${1:?}"` above to check if the variable is set but I like the explicit error in here 👍